services/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>life.qbic</groupId>
         <artifactId>parent-pom</artifactId>
-        <version>2.2.0</version>
+        <version>${revision}</version>
```

Comment:
where is `${revision}` set?

Reply:
in the `<properties>` section in the root pom of the project

Reply:
Oh, I overlooked that, thanks :)